infrastructure/variables.tf

```diff
@@ -55,8 +55,17 @@ variable "common_tags" {
   type = "map"
 }
 variable "is_frontend" {
-  default = false
```

Comment:
based on my latest knowledge (which can be easily outdated), all environments "is_frontend" property should be false other than the prod environment. Don't ask me why, Teo told me and this change fixed the builds once...